bool isQuery
---
int32[] ActuatorList
int32[] ActuatorSwitch